spec/serializers/move_serializer_spec.rb

```diff
@@ -129,6 +129,25 @@
     it 'contains an included from and to location' do
       expect(result[:included]).to(include_json(expected_json))
     end
+
+    context 'without a to_location' do
```

Comment:
What about making the context explicit, adding `to_location: nil` when creating the move?

Reply:
The only time a move can have a `to_location: nil` is when it's a prison recall, which is what is defined  here: https://github.com/ministryofjustice/hmpps-book-secure-move-api/pull/551/files#diff-5744dd109e1ed343bd07f744c7d8b00fR141